includes/Client.hpp

```diff
@@ -6,20 +6,28 @@
 class Client{
 	private:
 		int				_client_fd;
-		std::string		_readbuf;
-		std::string		_sendbuf;
-		bool			_to_deconnect;
+		char*			_readbuf;
+		char*			_sendbuf;
 		std::string		_nickname;
 		std::string		_username;
 		std::string		_realname;
 		std::string		_mode;
-		bool			_connection_password;
+		bool			_password;
 		bool			_registrationDone;
 		bool			_welcomeSent;
 	
 	public:
 	Client();
```

Comment:
Is there any situation where we would make an instance of client class without parameters? If not default constructor should be moved to private :)